utils/coap/coap-test-node.py

```diff
@@ -282,6 +284,44 @@ def render_put(self, request):
         return response
 
 
+class JavascriptResource(resource.Resource):
+    """Test node Javascript ressource."""
+
+    def __init__(self):
+        super(JavascriptResource, self).__init__()
+        self.script = """
+this.ledorange = saul.get_by_name("led");
```

Comment:
indentation looked weird to me at first, and then I noticed the triple quotes...

there is no perfect way to deal with multiline strings imo.
maybe [textwrap.dedent()](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) do you know it ?

Reply:
no I don't but thanks for pointing it. I think it's fine with the actual unindented triple quotes.